annlabmed.csl

```diff
@@ -0,0 +1,351 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal">
```

Comment:
Since this is (presumably) and English-only journal, please add `default-locale="en-US"`